src/css/jwplayer/flags/floatingplayer.less

```diff
@@ -26,17 +36,26 @@
 
 .jw-float-icon {
     display: none;
-
     position: absolute;
     right: 0;
     bottom: 0;
 }
 
-@keyframes jw-float {
+@keyframes jw-float-to-bottom {
     from {
         transform: translateY(100%);
     }
 
+    to {
+        transform: translateY(0);
+    }
+}
+
+@keyframes jw-float-to-top {
+    from {
+        transform: translateY(-100%);
+    }
+
```

Comment:
Why are we using an animation when we can just apply a transition on `transform`?

@radium-v Would you agree that we could just `transition` this `transform` , or is there a reason for using an animation?

Reply:
`transition` would require more class and event management in between the not-floated and floated states since they both sit at the origin (`0, 0`).

Essentially, `transition` doesn't work here because there's no value to transition from. `animation` allows for an immediate starting value when applied, regardless of any existing/nonexisting `transform` state.